### DISCUSSION TITLE
Is it possible to have a link does that not have a preview image attached to it?

### DISCUSSION THREAD

**Discussion by hilarytsai** [mozilla/hubs#3080] [General Questions & Discussion] [answered]
I need to link someone's space built in Unity, and the preview image for whatever reason, shows an error screen which we do not want to display as the image link to the space.  Is there a way to just have the link be blank, and perhaps overlay it on another image?  From what I can tell, all links must have preview images but I thought I'd ask if anyone knows a workaround for this.  The purpose is just to replace an unpalatable image accompanying the link.  Thank you!

**Comment by rawnsley** [comment, marked as answer]
You can overlay another image if you slip it between the preview image and the buttons. Here's an example https://hubs.mozilla.com/scenes/vSdw95m

**Reply by hilarytsai** [reply]
Thank you for this solution, @rawnsley!  I had a look and see that indeed, your link preview is not visible over the image. However, I seem to have trouble setting this up in my own test scene.  I see you put the link .001 in z space and the image is at 0 z position, so I did the same.  However, my link preview image still shows up. I think I understand what you mean by put it "between the preview and buttons, but I guess I'm just not sure why mine doesn't work when it's the exact same z position parameters as yours. If you wouldn't mind, could you take a look and see where mine went wrong?  I appreciate it!!

https://hubs.mozilla.com/scenes/JDhXTNy

**Reply by rawnsley** [reply]
Only a couple of issues with this scene:

1. This trick only works in one direction, so make sure you spawn at the "front" of the picture (indicated by the blue arrow in Spoke)
2. The Image must have controls disabled or it will steal focus
3. The image must be slightly (1mm or so) in front of the link, because you were spawning at the back you have placed it behind

This version has those changes and should work: https://hubs.mozilla.com/scenes/56gSiCn

**Reply by hilarytsai** [reply]
I see! Thank you rawnsley, I will disable my controls.  I did wonder if the avatar facing another direction has something to do with it. This is helpful to know!

**Comment by davegoopot** [comment]
I couldn't get this to work with an actual broken image. I tried e.g. https://soundcloud.com/redraemusic and it resulted in the image below. See that the broken image is already bleeding through, but the "open link" button won't pop up. The link is 0.001 units behind the image.

![image](https://user-images.githubusercontent.com/3875290/94606687-bd56df80-0292-11eb-9443-f5037e27ebab.png)
